client/src/services/courses.js

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import axios from './axios';
+
+const courses = async () => {
```

Comment:
Should probably be named getCourses since additional services for adding/modifying courses might be added.

Reply:
Changed.